Sources/PIRService/ReloadService.swift

```diff
@@ -24,9 +24,14 @@ struct ServerConfiguration: Codable {
         let shardCount: Int
         let versionCount: Int?
     }
+    
+    struct User: Codable {
```

Comment:
This does not represent a single user, so let's rename it.

```suggestion
    struct UserGroup: Codable {
```